lib/sejfguru/assets/location.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Sejfguru.Assets.Location do
+  @all_locations_label "Wszystkie miasta"
+
+  def city_lables do
+    Map.keys(locations().cities)
+  end
+
+  def other_labels do
+    Map.keys(locations().other)
+  end
+
+  def all_locations_label do
+    @all_locations_label
+  end
+
+  def locations do
+    %{
+      cities: %{
+        "Białystok" => ["Białystok"],
+        "Gdańsk" => ["Grunwaldzka 103A"],
+        "Kraków" => ["Pawia 9", "Krakow Warehouse"],
+        "Poznań" => ["Wojskowa 6", "Poznan Warehouse"],
+        "Warszawa" => ["Jana Pawla II 29", "Warszawa Warehouse"],
+        "Wrocław" => ["Wita Stwosza 15/9A"],
+        "Zielona Góra" => ["Zielona Góra"],
+      },
+      other: %{
+        "Pozostałe" => ["Remote"],
```

Comment:
the label (key) "Pozostałe" when used in navbar with locations is not very understandable, maybe we could use "Remote" as label as well?